XenModel/XenAPI-Extensions/VM.cs

```diff
@@ -678,12 +679,21 @@ public DateTime BodgeStartupTime
         void VirtualizationTimer_Elapsed(object sender, ElapsedEventArgs e)
         {
             NotifyPropertyChanged("virtualisation_status");
+            NotifyPropertyChanged("virtualisation_status_for_search");
         }
 
         private Timer VirtualizationTimer = null;
 
         [Flags]
-        public enum VirtualisationStatus { UNKNOWN = 0, OPTIMIZED = 1, PV_DRIVERS_NOT_INSTALLED = 2, PV_DRIVERS_OUT_OF_DATE = 4 };
+        public enum VirtualisationStatus
+        {
+            UNKNOWN                     = 1,
+            OPTIMIZED                   = 1 << 1, 

```

Comment:
I don't think we need to retain OPTIMIZED. We should use IO_DRIVERS_INSTALLED|MANAGEMENT_INSTALLED, and when we need to test for the presence of Tools, we should use whichever of those two we actually need.